include/openssl/crypto.h

```diff
@@ -245,6 +245,54 @@ typedef struct {
     struct CRYPTO_dynlock_value *data;
 } CRYPTO_dynlock;
 
+#if !defined(OPENSSL_THREADS) || defined(CRYPTO_TDEBUG)
+typedef int CRYPTO_MUTEX;
+typedef int CRYPTO_ONCE;
+typedef int CRYPTO_THREAD_LOCAL;
+typedef int CRYPTO_THREAD_ID;
+
+#  define CRYPTO_ONCE_STATIC_INIT 0
```

Comment:
That's on purpose. the define is indented a level deeper because it's inside the if. Looks like we use only one space indentation for #if, so I used that here as well.